src/components/Sidebar.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import '../styles/sidebar.styl';
```

Comment:
Please name this and other styles after components with capitalized first letter.